src/sql/workbench/contrib/notebook/browser/media/notebook.contribution.css

```diff
@@ -8,11 +8,16 @@
 	background-color: rgba(255, 255, 255, 0.4);
 }
 
-/* Activity Bar */
+/* Checked */
 .monaco-workbench .activitybar .monaco-action-bar .checked .action-label.book {
 	background-color: rgb(255, 255, 255);
 }
 
+/* Hovered */
+.monaco-workbench .activitybar .monaco-action-bar .action-label.book:hover {
+	background-color: rgba(255, 255, 255);
```

Comment:
I'm curious why this is even necessary since the sidebar contributions should all be done from the VS Code side. What happens if you remove these rules?

Reply:
I think it's because their icons come from codicons while we put our own explicitly, so it doesnt work for us. These rules def are needed for the styling, I tried it